src/blog/2023/02/flowforge-1-4-0-released.md

```diff
@@ -0,0 +1,99 @@
+description: FlowForge v1.4 is now available, FEATURE
+date: 2023-02-16 18:00:00.0
+authors: ["ian-skerrett"]
+video: 
+image: /blog/2023/01/images/flowforge-130-hero.png
+---
+
+New features include automatic device provisioning and support for staged development.
+
+<!--more-->
+
+We're pleased to announce version 1.4 is now available!  Keep reading for the details of what's in this release or you can watch our 1 minute roundup video of the new release above.
+
+To make it easy for everyone to experience FlowForge, we are introducing a new free 30-day trial. You can now experience the power of using FlowForge to quickly deliver Node-RED applications in a reliable, repeatable, collaborative, and secure manner. To get your free trial simply [sign up for FlowForge Cloud;](https://app.flowforge.com/account/create) no credit card is required!.
+
+## New User Features
+
+
+**Automatic Device Provisioning**
+
+FlowForge 1.4 now includes automatic device onboarding for fleets of devices. Simply download a FlowForge device provisioning credential to a new device or fleet of devices. On device restart, the FlowForge agent and the Node-RED snapshot will automatically be provisioned to the device. [Issue #1212](https://github.com/flowforge/flowforge/issues/1212)
+
+
+<div><iframe width="560" height="315" src="https://www.youtube.com/embed/B7XK3TUklUU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+**Support for Staged Development**
+
+A new feature of 1.4 is the ability to setup staged deployments. This makes it possible to simply move a project between a Development > Test > Production for your Node-RED application delivery. [Issue #1580](https://github.com/flowforge/flowforge/issues/1580)
+
+
+<div><iframe width="560" height="315" src="https://www.youtube.com/embed/JRk-Cf7eNIo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+
+## Improvements
+
+- It is now possible to change the project type. This includes changing the version of Node-RED being used for the project and the small/medium/large.  [#595](https://github.com/flowforge/flowforge/issues/595)
+
+- Show a preview of Team Library entries on click. [#1657](https://github.com/flowforge/flowforge/issues/1657)
+
+- Add a synchronous mode for the FlowForge context store. [#17](https://github.com/flowforge/flowforge-nr-persistent-context/issues/17)
+
+- Add a Last Seen status for devices connecting to FlowForge. [#1599](https://github.com/flowforge/flowforge/issues/1599)
+
+- Added a check to ensure the team slug is unique. [#1609](https://github.com/flowforge/flowforge/issues/1609)
+
+- Optionally set snapshot as target at creation [#1527](https://github.com/flowforge/flowforge/issues/1527)
+
```

Comment:
Agents will keep checking the forge endpoint, agents can now be provisioned without internet connection for example